todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskContract.java

```diff
@@ -41,7 +41,9 @@
 
         void createTask(String title, String description);
 
-        void updateTask( String title, String description);
+        void updateTask(String title, String description);
+
+        void onTaskFabClick(String title, String description);
```

Comment:
This name in the contract is referring to a detail in the view. The presenter shouldn't care about it being a FAB or a menu item. saveTask(...) would be better.